osie-runner/Dockerfile

```diff
@@ -6,22 +6,28 @@ VOLUME /statedir
 
 WORKDIR /tmp
 COPY requirements.txt .
-RUN apk update && apk add --update --upgrade --no-cache \
-           bash \
-           curl \
-           alpine-sdk \
-           docker \
-           jq \
-           libstdc++ \
-           linux-headers \
-           python3-dev
-# Latest version is ok/on purpose
-RUN  pip install -r requirements.txt  \
-    &&  python3 -m pip uninstall -y pip \
-    && apk del alpine-sdk linux-headers python3-dev \
-    && rm -rf /tmp/* $HOME/.cache
-WORKDIR /
+# runtime deps
+RUN apk add --update --upgrade --no-cache \
+        bash \
+        docker \
+        jq \
+        libstdc++ \
+        python3 \
```

Comment:
The base image is python:3.9, why reinstall it again?

Reply:
copy/paste

Reply:
fixed, ptal again.